lib/archethic/beacon_chain.ex

```diff
@@ -206,6 +209,19 @@ defmodule Archethic.BeaconChain do
     |> Enum.to_list()
   end
 
+  @doc """
+  Returns the current summary's replication attestations that current node have
+  """
+  @spec get_current_summary_replication_attestations(subset :: binary()) ::
+          list(ReplicationAttestation.t())
+  def get_current_summary_replication_attestations(subset) do
+    SummaryCache.stream_current_slots(subset)
+    |> Stream.flat_map(fn {%Slot{transaction_attestations: transaction_attestations}, _} ->
+      transaction_attestations
+    end)
+    |> Enum.to_list()
```

Comment:
Here you should aggregate the confirmations using the function `ReplicationAttestation.reduce_confirmations`